pkgs/top-level/all-packages.nix

```diff
@@ -18690,6 +18690,11 @@ with pkgs;
 
   xmacro = callPackage ../tools/X11/xmacro { };
 
+  xmlcopyeditor = callPackage ../applications/editors/xmlcopyeditor {
+    libxml = libxml2;
+    wxwidgets = wxGTK;
```

Comment:
Change the arguments `libxml`/`wxwidgets` to `libxml2`/`wxGTK` at the top of `default.nix` instead.